api/src/main/java/uk/gov/dhsc/htbhf/claimant/repository/ClaimRepository.java

```diff
@@ -42,6 +48,20 @@
     @Query("SELECT claim.id FROM Claim claim where claim.claimStatus = 'NEW'")
     List<UUID> getNewClaimIds();
 
+    /**
+     * Gets the claims that have a card_status of PENDING_CANCELLATION and a card_status_timestamp older than or equal to than the given period.
+     *
+     * @param period length of time the card status has been PENDING_CANCELLATION.
+     * @return list of claims
+     */
+    default List<Claim> getClaimsWithCardStatusPendingCancellationForMoreThan(Period period) {
+        LocalDateTime timestamp = LocalDateTime.now().minus(period);
+        return getClaimsWithCardStatusSinceBeforeGivenTimestamp(PENDING_CANCELLATION, timestamp);
+    }
+
+    @Query("SELECT claim FROM Claim claim where claim.cardStatus = :cardStatus and claim.cardStatusTimestamp <= :date")
+    List<Claim> getClaimsWithCardStatusSinceBeforeGivenTimestamp(@Param("cardStatus") CardStatus cardStatus, @Param("date") LocalDateTime localDateTime);
```

Comment:
Don't think you need the word `Since` in this method name, `getClaimsWithCardStatusBeforeGivenTimestamp` works pretty well.

Reply:
Removed since.